docs/gettinginvolved/acceptance-testing.rst

```diff
@@ -119,6 +120,7 @@ The configuration file for the Rackspace provider looks like:
      username: <rackspace username>
      key: <access key>
      keyname: <ssh-key-name>
+     # FLOC-1925 Add auth_url, auth_plugin, provider
```

Comment:
These extra pieces weren't necessary before because the acceptance test runner just ran the ZFS backend on a Rackspace VM, right?  And that didn't require using the OpenStack API - just libcloud with username and access key.  But now we want to run against the Cinder storage driver.  Does this imply that `run-acceptance-tests` will only support testing the "native" storage driver on each cloud provider?